### PR TITLE
[Dictionary] Update next repeat

### PR DESCRIPTION
**Pull request by livecodealice**
Put `Form:` on a new line and changed it to `**Form:**` for visibility.
Adjusted indentation of code example and removed empty line from it.

### CODE REVIEW

File: docs/dictionary/control_st/next-repeat.lcdoc [hunk 1]
```diff
@@ -25,10 +27,9 @@ example performs the loop action only if the <current card|current
 card's> name contains the letter "e" :
 
     repeat for the number of cards of this stack
```
Comment (montegoulding) [inline docs/dictionary/control_st/next-repeat.lcdoc]: @livecodeali @livecodesam what do you think about this example? Perhaps something different as there's definitely better ways to build a list of card names containing the letter `e`:

```
get the cardNames of this stack
filter lines of it with "*e*"
```

Comment (montegoulding) [reply docs/dictionary/control_st/next-repeat.lcdoc]: Also I suspect it would be a good policy for our examples not to use single line `if`

Comment (montegoulding) [reply docs/dictionary/control_st/next-repeat.lcdoc]: How about something common:
```
local tFolders
repeat for each line tFolder in folders(tPath)
    if tFolder begins with "." then
        next repeat
    end if
    put tPath & slash & tFolder & return after tFolders
end repeat
```

File: docs/dictionary/control_st/next-repeat.lcdoc [hunk 2]
```diff
@@ -24,12 +24,14 @@ anywhere inside a <repeat> <control structure>.
 The <next repeat> <control structure> skips the rest of the current
 <iteration>, and continues to the next <iteration>. The following
 example performs the loop action only if the <current card|current
-card's> name contains the letter "e" :
-
-    repeat for the number of cards of this stack
-        go to next card
-        if "e" is in the short name of this card then next repeat
-        put the short name of this card & return after myCardsList
+card's> name does not begin with the character ".":
```
Comment (livecodeali) [inline docs/dictionary/control_st/next-repeat.lcdoc]: This should be ...performs the loop action only if the folder name does not begin with the character "."

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 0ab4bc7

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 0ab4bc742cb98601141aed486919ca601d062549

**Comment by livecode-vulcan** [comment]
😎 test success 0ab4bc742cb98601141aed486919ca601d062549

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/742)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/758)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/732)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/675)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/677)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/736)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/756)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/741)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/758)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/740)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/751)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/772)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/736)

**Pull request merged**